Bindings/Java/Matlab/Hopper_Device/EvaluateHopper.m

```diff
@@ -24,7 +24,7 @@
 % Perform a hop with the provided model and evaluate the performance of the
 % hop.
 
-function [score, peakHeight, finalHeight] = EvaluateHopper(hopper, print)
+function [score, peakHeight, finalHeight, heightStruct] = EvaluateHopper(hopper, visualize, print)
```

Comment:
I agree. I left in just in case we want to add any other penalties later. In fact, if we just keep the only metric as `peakHeight`, we can just return `peakHeight` and `heightStruct` (and I can just pull the `finalHeight` out for my own use if needed). I think this can be deferred to a later commit.